test/corpus/arguments.txt

```diff
@@ -205,16 +205,17 @@ EXTERNAL *netrw-externapp* {{{2
   (block
     (line
       (h1
-        (word)
-        (word)
+        (separator)
+        (heading
+          (word)
+          (word))
```

Comment:
nice improvement. though i think the separator previously was just not captured, is it useful to capture it ?

Reply:
It was not exposed, correct. I think it's useful if people want to give it a different highlight group and -- especially -- conceal (replace with proper line-drawing character).